src/index.js

```diff
@@ -0,0 +1,170 @@
+"use strict";
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    increaseTempControl: null,
+    decreaseTempControl: null,
+    tempValue: null,
+    landscape: null,
+    headerCityName: null,
+    cityNameInput: null,
+    currentTempButton: null,
+    temp: tempValue.textContent,
+    skySelect: null,
+    sky: null,
+    cityNameReset: null,
+    gardenContent: null,
+};
+
+const loadControls = () => {
```

Comment:
Interesting approach! Very creative way to retrieve your state values all at once!